scripts/recan_bundle_tdr.py

```diff
@@ -109,8 +112,9 @@ def content_length(content: JSON) -> int:
     return len(json.dumps(content).encode('UTF-8'))
 
 
-def random_uuid() -> str:
-    return str(uuid.uuid4())
+def deterministic_uuid(bundle_uuid: str, entity_name: str) -> str:
+    namespace = uuid.UUID(bundle_uuid)
+    return str(uuid.uuid5(namespace, entity_name))
```

Comment:
For future reference: At this point it would help if the bundle uuid was contextual but that can only be done by converting this to an class so this would read:

```suggestion
def deterministic_uuid(self, entity_name: str) -> str:
    namespace = uuid.UUID(self.bundle_uuid)
    return str(uuid.uuid5(namespace, entity_name))
```

But leave it as is.